src/components/opoTable/opoTable.js

```diff
@@ -14,88 +16,137 @@ export default function OpoTable({
   opos,
   title,
 }) {
-  if (!opos.length) return null;
+  const columns = useMemo(() => {
+    const cols = inState
+      ? ["name", "region", "tier", "donors", "shadow", "investigation"]
+      : ["states", "name", "tier", "donors", "shadow"];
 
-  const Heading = ({ color, title }) => (
-    <th scope="col" className={color ?? null}>
-      {headings[title].heading}
-      {citations[title] && (
-        <sup>
-          <a
-            className={color ?? null}
-            href={`#citations-${citations[title].index}`}
-            target="_self"
-          >
-            {citations[title].index + 1}
-          </a>
-        </sup>
-      )}
-    </th>
-  );
+    const Heading = ({ color, title }) => (
+      <div>
+        {headings[title].heading}
+        {citations[title] && (
+          <sup>
+            <a
+              className={color ?? null}
+              href={`#citations-${citations[title].index}`}
+              target="_self"
+            >
+              {citations[title].index + 1}
+            </a>
+          </sup>
+        )}
+      </div>
+    );
+    const createCol = accessor => {
+      let col = {
+        Header: (
+          <Heading
+            title={accessor}
+            color={accessor === "shadow" ? "red" : null}
+          />
```

Comment:
nit: if it's just passed here no reason to have the standalone `Heading` component--can just be passed into the `Header` prop?